backend/api/Areas/Keycloak/Controllers/RoleController.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using Entity = Pims.Dal.Entities;
+using Model = Pims.Api.Areas.Keycloak.Models;
+using Pims.Api.Policies;
+using Pims.Dal.Security;
+using Pims.Dal.Keycloak;
+
+namespace Pims.Api.Areas.Keycloak.Controllers
+{
+    /// <summary>
+    /// RoleController class, provides endpoints for managing the integration between keycloak groups and PIMS roles.
+    /// </summary>
+    [HasPermission(Permissions.AdminRoles)]
+    [ApiController]
+    [Area("keycloak")]
+    [Route("/api/[area]/roles")]
+    public class RoleController : ControllerBase
```

Comment:
Replaced `GroupController` with `RoleController`.  This means the endpoint changed to `/api/keycloak/roles/`